pkg/skaffold/sync/sync.go

```diff
@@ -108,3 +112,44 @@ func intersect(context string, syncMap map[string]string, files []string) (map[s
 	}
 	return ret, nil
 }
+
+func Perform(ctx context.Context, image string, files map[string]string, cmdFn func(context.Context, v1.Pod, v1.Container, string, string) *exec.Cmd) error {
```

Comment:
You can still leave this as unexported, I dont think we want it used outside the package.

Reply:
so the `Sync` function in the kubectl package calls it, so I think it has to be exported?

Reply:
doh you are correct, LGTM then